google/resource-snippets/bigquery-v2/README.md

```diff
@@ -0,0 +1,33 @@
+# Bigquery Snippets
+
```

Comment:
could you add a short setup section mentioning which tools are required? from a quick scan I saw

* `gcloud`
* `gsutil`
* `kubectl`
* `terraform`
* `kpt`